domain/cells/src/commonMain/kotlin/com/wire/kalium/cells/domain/CellsRepository.kt

```diff
@@ -74,4 +74,5 @@ internal interface CellsRepository {
     suspend fun clearPublicLinkPassword(linkUuid: String)
     suspend fun setPublicLinkExpiration(linkUuid: String, expiresAt: Long?): Either<NetworkFailure, Unit>
     suspend fun getNodeVersions(uuid: String): Either<NetworkFailure, List<NodeVersion>>
+    suspend fun restoreNodeVersion(uuid: String, versionId: String): Either<NetworkFailure, Unit>
```

Comment:
This function could be in an `internal fun interface NodeVersionRestorer`, no need to make the CellsRepository have more and more functions.

It would make testing easier, and dependencies more straightforward without having to maintain a huge `Fake` repository.